src/sgame/sg_buildable.cpp

```diff
@@ -1702,6 +1702,23 @@ itemBuildError_t G_CanBuild( gentity_t *ent, buildable_t buildable, int /*distan
 		level.numBuildablesForRemoval = 0;
 	}
 
+	int max_miners = g_maxMiners.Get();
+	if ( max_miners >= 0 && ( buildable == BA_H_DRILL || buildable == BA_A_LEECH ) )
```

Comment:
I really think `if ( max_miners > 0` would be better here, no need to do anything if no miner is allowed anyway.

Reply:
If we do nothing if no miner is allowed, we do in fact allow an infinite number of miners.